targets/targets.json

```diff
@@ -3260,7 +3260,7 @@
             "MX25LM51245G"
         ],
         "components_add": [
-            "FLASHIAP"
+            "OSPIF"
```

Comment:
You can't remove FLASHIAP
Regards

Reply:
> You can't remove FLASHIAP
> Regards

Hi,

I removed it just for only testing OSPI. I have added it now.